src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,497 @@
+    type: command
+    short-summary: check for the availability of the given name for the Namespace
+    examples:
+        - name: Create a new topic.
+          text: az servicebus namespace check_name_availability --name mynamespace
+
+    """
+
+helps['servicebus namespace create'] = """
+    type: command
+    short-summary: Creates the ServiceBus Namespace
+    examples:
+        - name: Create a new namespace.
+          text: helps['az servicebus namespace create --resource-group myresourcegroup --name mynamespace --location westus
+           --tags ['tag1: value1', 'tag2: value2'] --sku-name Standard --sku-tier Standard']
+
+    """
+
+helps['servicebus namespace show'] = """
+    type: command
+    short-summary: shows the Namespace Details
+    examples:
+        - name: shows the Namespace details.
+          text: helps['az servicebus namespace show --resource-group myresourcegroup --name mynamespace']
+
+    """
+
+helps['servicebus namespace list'] = """
+    type: command
+    short-summary: List the Namespaces by ResourceGroup or By subscription
+    examples:
+        - name: Get the Namespaces by resource Group.
+          text: helps['az servicebus namespace list --resource-group myresourcegroup']
+        - name: Get the Namespaces by Subscription.
+          text: helps['az servicebus namespace list']
+
+    """
+
+helps['servicebus namespace delete'] = """
+    type: command
+    short-summary: Deletes the Namespaces
+    examples:
+        - name: Deletes the Namespace
+          text: helps['az servicebus namespace delete --resource-group myresourcegroup --name mynamespace']
+
+    """
+
+helps['servicebus namespace authorizationrule create'] = """
+    type: command
+    short-summary: Creates Authorization rule for the given Namespace
+    examples:
+        - name: Creates Authorization rules
+          text: helps['az servicebus namespace authorizationrule create --resource-group myresourcegroup --namespace-name mynamespace
+           --name myauthorule --access-rights [Send, Listen]']
+
+    """
+
+helps['servicebus namespace authorizationrule get'] = """
+    type: command
+    short-summary: Shows the details of AuthorizatioRule
+    examples:
+        - name: Shows the details of AuthorizatioRule
+          text: helps['az servicebus namespace authorizationrule show --resource-group myresourcegroup --namespace-name mynamespace
+           --name myauthorule']
+
+    """
+
+helps['servicebus namespace authorizationrule list'] = """
+    type: command
+    short-summary: Shows the list of AuthorizatioRule by Namespace
+    examples:
+        - name: Shows the list of AuthorizatioRule by Namespace
+          text: helps['az servicebus namespace authorizationrule show --resource-group myresourcegroup --namespace-name mynamespace']
+
+    """
+
+helps['servicebus namespace authorizationrule list-keys'] = """
+    type: command
+    short-summary: Shows the connectionstrings of AuthorizatioRule for the namespace
```

Comment:
`connectionstrings` should be two words not one.

Reply:
Resolved